$OpenBSD: patch-automoc4_files_in,v 1.1.1.1 2013/02/02 09:42:07 zhuk Exp $
Automoc could not parse Boost headers using BOOST_JOIN() macro,
so use this workaround until (if ever) Automoc gets fixed.
Patch taken from KDE bug 304111:
https://bugs.kde.org/show_bug.cgi?id=304111
--- automoc4.files.in.orig	Fri Nov 23 17:51:04 2012
+++ automoc4.files.in	Fri Nov 23 17:52:14 2012
@@ -1,9 +1,9 @@
 SOURCES:
 @_moc_files@
 MOC_COMPILE_DEFINITIONS:
-@_moc_cdefs@
+@_moc_cdefs@;BOOST_TT_HAS_OPERATOR_HPP_INCLUDED
 MOC_DEFINITIONS:
-@_moc_defs@
+@_moc_defs@ -DBOOST_TT_HAS_OPERATOR_HPP_INCLUDED
 MOC_INCLUDES:
 @_moc_incs@
 CMAKE_INCLUDE_DIRECTORIES_PROJECT_BEFORE:
